src/containers/SponsorLogos.js

```diff
@@ -1,10 +1,48 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
+import { db } from '../utility/firebase'
+import { DB_COLLECTION, DB_HACKATHON } from '../utility/Constants'
+import Sponsors from '../components/SponsorLogos'
 
-const CenteredH1 = styled.h1`
+const CenteredH1 = styled.h2`
   text-align: center;
+  margin-bottom: 1em;
 `
+const getSponsors = () => {
+  return db
+    .collection(DB_COLLECTION)
+    .doc(DB_HACKATHON)
+    .collection('Sponsors')
+    .get()
+    .then(querySnapshot => {
+      return querySnapshot.docs
+    })
+}
```

Comment:
if this function is only fetching the firebase stuff, can probably move it to the firebase utils file